core/arch/arm/kernel/thread.c

```diff
@@ -883,6 +888,20 @@ static void init_thread_stacks(void)
 }
 #endif /*CFG_WITH_PAGER*/
 
+static void init_user_kcode(void)
+{
+#ifdef CFG_CORE_UNMAP_CORE_AT_EL0
+	vaddr_t v;
+
+	v = (vaddr_t)thread_vect_table;
+	thread_user_kcode_va = ROUNDDOWN(v, CORE_MMU_USER_CODE_SIZE);
+	thread_user_kcode_size = CORE_MMU_USER_CODE_SIZE * 2;
```

Comment:
why 2 regions (likely pages)?

Reply:
Please make these comments in https://github.com/OP-TEE/optee_os/pull/2048 instead